src/HealthChecks.AzureServiceBus/AzureServiceBusTopicHealthCheck.cs

```diff
@@ -50,10 +41,14 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                 }
 
                 var message = new Message(Encoding.UTF8.GetBytes(TEST_MESSAGE));
-                
-                _configuringMessage?.Invoke(message);
+                if (RequiresSession)

```

Comment:
Well, adding session here is a new feature comparing with old code but this PR have a lo of changes on others healthchecks that does not change any functioanlity!

Reply:
I'm bringing it in parity with queues as they are identical and same service.

It's part of the same core (`AzureServiceBusHealthCheck`) but if you really want I can get it out and hope it doesn't impact too much.